app/views/layouts/_sub_menu.html.erb

```diff
@@ -25,8 +25,11 @@
 <% elsif @current_user.ta? && (controller.controller_name == 'assignments' ||
```

Comment:
instead of checking @current_user.ta? (and @current_user.admin? above) we should be using your new policies